packages/sketchapp-parser/src/lib/sketch-style-parser.service.ts

```diff
@@ -65,6 +64,27 @@ export class SketchStyleParserService {
     }
   }
 
+  autoFixPagePosition(page: SketchMSPage) {
```

Comment:
why did we move this from the UI store?

Reply:
Because the page position is translated to CSS by the parser: https://github.com/xlayers/xlayers/blob/0ab2d1df9a8d862eddf2e0366b003d0306f5bf1d/packages/sketchapp-parser/src/lib/sketch-style-parser.service.ts#L137
Or we have to override this CSS  rule directly into the viewer.